config.json

```diff
@@ -1338,7 +1354,13 @@
         "name": "Alphametics",
         "uuid": "a602bd33-69fc-4b67-a3d3-95198853095e",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "array-analysis",
+          "array-transformations",
+          "conditionals",
+          "for-loops",
+          "functions"
```

Comment:
Can you show me a solution that works with just those? Going by the example solution I would add `rest-and-spread`, `arithmetic-operators` and `objects`.